### DISCUSSION TITLE
Откат

### DISCUSSION THREAD

**Discussion by lbtboy** [AdguardTeam/AdGuardHome#4624] [Q&A]
Здравствуйте. Как правильно без потери данных файла yaml откатится c AdGuard Home v0.108.0-b.7 на стабильную версию AdGuard Home v0.107.6. Где-то находил что нужно в файле yaml что-то менять, но не могу найти в данный момент эту информацию. Заранее благодарен.

**Comment by ainar-g** [comment]
Добрый день.  Описание изменений в конфигурационном файле и процедур их отката всегда находятся в нашем [журнале изменений][1], в разделе _«Configuration changes»_ каждого релиза.

Но сразу хочу сказать, что уже довольно скоро мы планируем релиз версии v0.107.7, куда эти изменения скорее всего войдут, так что возможно, особого смысла что-то откатывать сейчас нет.

[1]: https://github.com/AdguardTeam/AdGuardHome/blob/master/CHANGELOG.md

**Comment by lbtboy** [comment]
Спасибо. Вас понял, конечно подожду. Я правильно понимаю можно будет AdGuard Home v0.108.0-b.7 обновить на v0.107.7 просто заменой файлов AdGuardHome, AdGuardHome.sig, CHANGELOG.md, LICENSE.txt, README.md и всё запустится без проблем?

**Reply by ainar-g** [reply]
Да.  По сути, заменить можно всё, оставив только `AdGuardHome.yaml`, `leases.db` и `data/`.

**Comment by lbtboy** [comment]
Здравствуйте. Обновил AdGuardHome, AdGuardHome.sig, CHANGELOG.md, LICENSE.txt, README.md на v0.107.7 c v0.108.0-b.7.

При запуске команды nslookup на выходе получаю:
;; Warning: query response not set
;; Got recursion not available from 127.0.0.1, trying next server

При запуске apt update имею на выходе:
Ошб:1 http://security.debian.org/debian-security bullseye-security InRelease
  Не удалось разрешить «security.debian.org»
Ошб:2 http://deb.debian.org/debian bullseye InRelease
  Не удалось разрешить «deb.debian.org»
Ошб:3 http://deb.debian.org/debian bullseye-updates InRelease
  Не удалось разрешить «deb.debian.org»
Ошб:4 http://deb.debian.org/debian bullseye-backports InRelease
  Не удалось разрешить «deb.debian.org»
Чтение списков пакетов… Готово
Построение дерева зависимостей… Готово
Чтение информации о состоянии… Готово         
Все пакеты имеют последние версии.
W: Не удалось получить http://deb.debian.org/debian/dists/bullseye/InRelease  Не удалось разрешить «deb.debian.org»
W: Не удалось получить http://deb.debian.org/debian/dists/bullseye-updates/InRelease  Не удалось разрешить «deb.debian.org»
W: Не удалось получить http://deb.debian.org/debian/dists/bullseye-backports/InRelease  Не удалось разрешить «deb.debian.org»
W: Не удалось получить http://security.debian.org/debian-security/dists/bullseye-security/InRelease  Не удалось разрешить «security.debian.org»
W: Некоторые индексные файлы скачать не удалось. Они были проигнорированы, или вместо них были использованы старые версии.
 
При возврате на  v0.108.0-b.7 все резолвится всё работает. При обновлении на v0.108.0-b.9 ситуация становится как  c версией v0.107.7 ни чего не работает.

**Reply by ainar-g** [reply]
@lbtboy, добрый день.  Надо смотреть [подробные логи сервиса][1].  Если из них будет непонятно, в чём дело, можете прислать их нам на devteam@adguard.com.

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Comment by lbtboy** [comment]
Понял конечно c чем это связано, но не понял почему. Ситуация такая один из вышестоящих резолверов не отвечает quic://dns.comss.one:784, я сменил VPS и возможно IP у них в стоп листе данного провайдера, они это практикуют, удаление не отвечающего резолвера приводит службу в норбу без сбоев.. Вот отсюда и вопрос почему в версии v0.107.7 такая ситуация, не отвечает один вышестоящий резолвер перестает работать вся служба AdGuardHome не резолвит ни один DNS. На версии  v0.108.0-b.7, v0.107.6 нет такой проблемы. Я бы этого и не выяснил если бы и на домашнем роутере не появилась такая же проблема после обновления c v0.107.6 на v0.107.7 я добавил tls://getdnsapi.net и он по счастливой случайности почему-то не резолвится c моего домашнего IP, удалил  tls://getdnsapi.net все работает как атомные часы. Вот такая случайность выявила данный баг, хотя может это и не баг, а надо что  то просто отключить. Надеюсь на Вашу помощь, если нужны будут подробные логи скажите как включить и сколько по времени собирать.

**Reply by ainar-g** [reply]
@lbtboy, мы писали про это в журнале изменений.  AdGuard Home теперь реализует исходящий DNS-over-QUIC по финальной версии стандарта RFC 9250.  У многих провайдеров всё ещё используется старая версия черновика стандарта.  См. #4644.

В качестве временных решений можете либо использовать провайдер, который реализует финальную версию стандарта, либо перейти с DNS-over-QUIC на DNS-over-HTTPS или DNS-over-TLS, которые являются более стабильными.

**Comment by lbtboy** [comment]
Я бы c превеликим удовольствием согласился со всем этим, если бы не терялось соединение c интернет, потому что кроме неработающего к примеру quic://dns.comss.one:784, не было прописано больше ни одного DNS. И более того я Вам скажу что на домашнем роутере  c прописанным quic://dns.comss.one:784 он работает прекрасно, но стоит прописать неработающий, сеть ложится как и на удаленном VPS.
